bsdlib/include/nrf_socket.h

```diff
@@ -724,6 +739,59 @@ typedef struct
     nrf_gnss_agps_data_tow_element_t sv_tow[NRF_GNSS_AGPS_MAX_SV_TOW]; /**< TOW assistance data for PRN n */
 } nrf_gnss_agps_data_system_time_and_sv_tow_t;
 
+typedef struct
+{
+    int32_t latitude;          /**< Geodetic latitude in WGS-84. Range -8388607...8388607.
+                                *   The relation between the coded number N and the latitude
+                                *   range X (in degrees) is as follows: N <= (2^24/360) * X < N + 1.
+                                *   For N = 2^23 - 1, the range is extended to include N+1.
+                                *   Range of X (in degrees) -90...90.
+                                */
+
+    int32_t longitude;         /**< Geodetic longitude in WGS-84. Range -8388607..-8388607.
```

Comment:
`Range -8388607..-8388607.` The second value should be positive.

Reply:
Good catch, fixed.